bls/src/test/java/tech/pegasys/teku/bls/impl/mikuli/MikuliPublicKeyTest.java

```diff
@@ -47,22 +47,36 @@ void succeedsWhenTwoInfinityPublicKeysAreEqual() {
 
   @Test
   void succeedsIfDeserializationOfInfinityPublicKeyIsCorrect() {
-    BLSPublicKey infinityPublicKey = new BLSPublicKey(new MikuliPublicKey(new G1Point()));
+    MikuliPublicKey infinityPublicKey = new MikuliPublicKey(new G1Point());
     byte[] pointBytes = new byte[48];
     pointBytes[0] = (byte) 0xc0;
     Bytes infinityBytesSsz =
         SSZ.encode(
             writer -> {
               writer.writeFixedBytes(Bytes.wrap(pointBytes));
             });
-    BLSPublicKey deserializedPublicKey = BLSPublicKey.fromBytes(infinityBytesSsz);
+    MikuliPublicKey deserializedPublicKey = MikuliPublicKey.fromBytesCompressed(infinityBytesSsz);
     assertEquals(infinityPublicKey, deserializedPublicKey);
   }
 
   @Test
   void succeedsWhenRoundtripSSZReturnsTheInfinityPublicKey() {
-    BLSPublicKey publicKey1 = new BLSPublicKey(new MikuliPublicKey(new G1Point()));
-    BLSPublicKey publicKey2 = BLSPublicKey.fromBytes(publicKey1.toBytes());
+    MikuliPublicKey publicKey1 = new MikuliPublicKey(new G1Point());
+    MikuliPublicKey publicKey2 =
+        MikuliPublicKey.fromBytesCompressed(publicKey1.toBytesCompressed());
     assertEquals(publicKey1, publicKey2);
   }
+
+  @Test
+  void infinityPublicKeyIsValid() {
+    MikuliPublicKey infinityG1 =
+        MikuliPublicKey.fromBytesCompressed(
+            Bytes48.fromHexString(
+                "0x"
+                    + "c0000000000000000000000000000000"
+                    + "00000000000000000000000000000000"
+                    + "00000000000000000000000000000000"));
+    infinityG1.forceValidation();
+    System.out.println(infinityG1.g1Point());
```

Comment:
Probably shouldn't have a `System.out.println` here and should assert something instead?